src/components/alert/alert.stories.js

```diff
@@ -126,6 +126,4 @@ storiesOf('Alert', module)
     notes: { markdown: notes }
   })
   .add(...makeStory('default', dlsThemeSelector))
-  .add(...makeStory('classic', classicThemeSelector, true))
-  .add(...makeButtonStory('with button', dlsThemeSelector))
-  .add(...makeButtonStory('with button classic', classicThemeSelector, true));
+  .add(...makeButtonStory('with button', dlsThemeSelector));
```

Comment:
@DlgSHi - That line exists here now though. It hasn't been removed from the code, just moved I think.